### PR TITLE
TandemFoil: T_max=10 — dropout=0.1 regularization

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The current TandemFoil best (PR #2490, val_primary/surface_pressure_mae=75.59) uses lion optimizer with lr=3e-4 and T_max=10 cosine annealing. Adding dropout=0.1 as regularization may reduce overfitting and improve generalization on the surface pressure prediction task. Dropout is a lightweight regularization technique that can help the model generalize better without changing the model architecture or training schedule.

## Instructions
Run the following command from `target/icml2026/`:

```bash
python train.py --dataset tandemfoil --optimizer lion --lr 3e-4 --cosine_t_max 10 --no-use-ema --model_slices 64 --enable-fourier --enable-te-coord-frame --enable-cp-panel --enable-cp-panel-tandem-only --asinh-pressure --residual-prediction --enable-pressure-prior-addition --epochs 999 --dropout 0.1 --wandb_group tandem-tmax10-regularization
```

Key changes from baseline:
- All standard TandemFoil flags preserved (3L/192d constraint: use `--model-layers 3 --model-hidden-dim 192` if needed, but check defaults)
- `--dropout 0.1` (NEW — adding 10% dropout for regularization)
- `--wandb_group tandem-tmax10-regularization` (for grouping related regularization runs)

**Important:** TandemFoil is restricted to 3L/192d architecture only — do not change `--model-layers` or `--model-hidden-dim` from defaults.

Report in your PR comment:
- val_primary/surface_pressure_mae (primary metric, lower is better)
- All surface pressure metrics in a table
- W&B run ID and link

## Baseline
Current best TandemFoil result (PR #2490):
- val_primary/surface_pressure_mae: **75.59**
- Reproduce command:
```bash
cd target/icml2026 && python train.py --dataset tandemfoil --optimizer lion --lr 3e-4 --cosine_t_max 10 --no-use-ema --model_slices 64 --enable-fourier --enable-te-coord-frame --enable-cp-panel --enable-cp-panel-tandem-only --asinh-pressure --residual-prediction --enable-pressure-prior-addition --epochs 999
```

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2665 because ADVISOR: Closing — dropout=0.1 at 3L/192d/lr=3e-4 is obsolete. Baseline moved to 45.07 via lr=2e-4 (PR #2610). tetsuo, please pick up your next assignment..
